src/test/java/org/folio/dcb/service/LendingLibraryServiceTest.java

```diff
@@ -45,43 +43,33 @@ class LendingLibraryServiceTest {
   private UserServiceImpl userService;
   @Mock
   private RequestServiceImpl requestService;
-  @Mock
-  private TransactionMapper transactionMapper;
 
   @Mock
   private CirculationServiceImpl circulationService;
 
   @Test
   void createTransactionTest() {
     var item = createDcbItem();
-    var patron = createDcbPatron();
+    var patron = createDefaultDcbPatron();
     var user = createUser();
 
-    when(transactionRepository.existsById(DCB_TRANSACTION_ID)).thenReturn(false);
+//    when(transactionRepository.existsById(DCB_TRANSACTION_ID)).thenReturn(false);
```

Comment:
I think it is better to remove all the commented lines if it is not needed

Reply:
done